packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -333,6 +345,30 @@ contract BridgePool is Testable {
                 .rawValue;
     }
 
+    function _getRealizedFeeAmount(uint64 _realizedFeePct, uint256 _amount) private pure returns (uint256) {
```

Comment:
this computation will likely chance in the next few days but it follows the interface spec for now.